sphinx/writers/latex.py

```diff
@@ -601,6 +601,11 @@ def __init__(self, document, builder):
                     (',numfigreset=%s' % self.numfig_secnum_depth)
             else:
                 self.elements['sphinxpkgoptions'] += ',nonumfigreset'
+            try:
+                if builder.config.math_numfig:
+                    self.elements['sphinxpkgoptions'] += ',mathnumfig'
+            except:
+                pass
```

Comment:
@tk0miya can you please check that. There is probably better way to access `math_numfig`, but I ended up using the try clause as a workaround.

Reply:
I have corrected the ``except``. But I would prefer more direct way to access this config value from the mathbase extension, if possible.